### PR TITLE
added number of library items as a tooltip

### LINKED ISSUE #10208: number of library items not fully displayed -> show as tooltip
### JabRef version

5.9 (latest release)

### Operating system

Windows

### Details on version and operating system

Windows 10 22H2

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

I have downloaded 2 databases in BibTeX format: one is an arcade game database (45974 items), the other a game console/computer database (134352)

Please look at screenshot. The number of items in the library is not fully displayed (see at the left). Only 4 digits are displayed, the rest in front of it is gone. Could it be fixed so I see the currect number of items? I know it are A LOT but it's possible to have such a big library :)

afbeelding

afbeelding

Attached are the 2 bib files

[games.zip](https://github.com/JabRef/jabref/files/12431909/games.zip)

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

### PR DESCRIPTION
**Pull request by DohaRamadan**
added the number of library items fully displayed as a tooltip
Fixes #10208
before:
before
after:
after

### Mandatory checks
- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by DohaRamadan** [comment]
Can someone help me understand why BibtexDatabaseWriterTest tests are failing?

**Comment by Siedlerchr** [comment]
Oh they are not related, they are coming from this commit https://github.com/JabRef/jabref/commit/c385eb442f07623d34ea62b6b58656080106b73c

**Comment by DohaRamadan** [comment]
Should i do something regarding my pull request or leave it as it is?

**Comment by Siedlerchr** [comment]
Add a changelog entry :)

**Comment by ThiloteE** [comment]
Since this is a UI related change, it would be nice to post screenshots from "before" and "after".

**Comment by DohaRamadan** [comment]
Done👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added support for the Finnish language. [#10183](https://github.com/JabRef/jabref/pull/10183)
 - We added the option to automatically replaces illegal characters in the filename when adding a file to JabRef. [#10182](https://github.com/JabRef/jabref/issues/10182)
 - We added a privacy policy. [#10064](https://github.com/JabRef/jabref/issues/10064)
+- We added the number of library items as a tooltip. [#10208](https://github.com/JabRef/jabref/issues/10208)
```
Comment (Siedlerchr) [inline CHANGELOG.md:46]:
```suggestion
- We added a tooltip to show the number of entries in a group [#10208](https://github.com/JabRef/jabref/issues/10208)
```

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**